### PR TITLE
[cloud-provider-vcd][discovery] Support multiple VCD API versions

### PR DESCRIPTION
**Pull request by asviel**
## Description

Support multiple VCD API versions.

## Why do we need it, and what problem does it solve?

We want to support the online volume expansion, which is available since vcd-csi-driver version 1.6.0, but we need to support an older version of the VCD API (< 37.2), which is not compliant with 1.6.0 vcd-csi-driver.

Compatibility matrix: https://github.com/vmware/cloud-director-named-disk-csi-driver/blob/15142e09f9ac1dfcb4860e3f307d28f092b947de/README.md

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cloud-provider-vcd
type: feature
summary: Support multiple VCD API versions.
impact_level: default
```

### CODE REVIEW

**Review by aleksey-su — APPROVED** [review]

File: ee/candi/cloud-providers/vcd/openapi/cloud_discovery_data.yaml [hunk 1]
```diff
@@ -44,3 +44,16 @@ apiVersions:
             isDefaultStorageProfile:
               type: boolean
         description: List of storage classes.
+      version:
```
Comment (RomanenkoDenys) [inline ee/candi/cloud-providers/vcd/openapi/cloud_discovery_data.yaml:47]:
```suggestion
      vcdApi:
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
Please discuss openapi naming with Maksim

**Comment by z9r5** [comment]
Please note: I've changed milestone to 1.61

**Review by name212 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

File: ee/candi/cloud-providers/vcd/openapi/cloud_discovery_data.yaml (hunk 1 above)
Comment (asviel) [reply ee/candi/cloud-providers/vcd/openapi/cloud_discovery_data.yaml:47]: It's not necessary for internal values.

**Comment by z9r5** [comment]
> Please discuss openapi naming with Maksim

Discussed at a meeting.

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8527](https://github.com/deckhouse/deckhouse/pull/8527) to the branch [release-1.60](https://github.com/deckhouse/deckhouse/tree/release-1.60) successful!

**Pull request merged**